labelbox/data/annotation_types/feature.py

```diff
@@ -24,3 +21,11 @@ def must_set_one(cls, values):
                 "Must set either feature_schema_id or name for all feature schemas"
             )
         return values
+
+    def dict(self, *args, **kwargs):
+        res = super().dict(*args, **kwargs)
+        if 'name' in res and res['name'] is None:
+            res.pop('name')
+        if 'featureSchemaId' in res and res.featureSchemaId is None:
```

Comment:
is there a reason we use `res['name']` and `res.featureSchemaId`?

Reply:
ah forgot to change it